docs/OV_Runtime_UG/ONNX_Support.md

```diff
@@ -0,0 +1,65 @@
+# ONNX Format Support {#ONNX_Format_Support}
+
+
+Since the 2020.4 release, OpenVINO™ has supported native usage of ONNX models. The `core.read_model()` method, which is the recommended approach to reading models, provides a uniform way to work with OpenVINO IR and ONNX formats alike. Example:
+
+@sphinxdirective
+.. tab:: C++
+
+   .. code-block:: cpp
+   
+      ov::Core core;
+      std::shared_ptr<ov::Model> model = core.read_model("model.xml")
+ 
+.. tab:: Python
+
+   .. code-block:: python
+
+      import openvino.runtime as ov
+      core = ov.Core()
+      model = core.read_model("model.xml")
+@endsphinxdirective
+
```

Comment:
```suggestion

While ONNX models are directly supported by OpenVINO™, it can be useful to convert them to IR format to take advantage of advanced OpenVINO optimization tools and features. For information on how to convert an ONNX model to the OpenVINO IR format, see the [Converting an ONNX Model](https://github.com/openvinotoolkit/openvino/pull/MO_DG/prepare_model/convert_model/Convert_Model_From_ONNX.md) page.
```